pages/mesosphere/dcos/services/beta-jenkins/4.0.0-2.204.2-beta/jenkins-auth/index.md

```diff
@@ -0,0 +1,278 @@
+   ```bash
+   
+    dcos security org users grant jenkins-principal dcos:mesos:master:framework:role:* create
+    dcos security org users grant jenkins-principal dcos:mesos:master:reservation:role:* create
+    dcos security org users grant jenkins-principal dcos:mesos:master:volume:role:* create
+    
+    dcos security org users grant jenkins-principal dcos:mesos:master:task:user:nobody create
+    dcos security org users grant jenkins-principal dcos:mesos:agent:task:user:nobody create
+
+    dcos security org users grant jenkins-principal dcos:mesos:master:reservation:principal:jenkins-principal delete
+    dcos security org users grant jenkins-principal dcos:mesos:master:volume:principal:jenkins-principal delete
+   ```
+
+# Create a config.json file
+
+The contents of the `config.json` file will vary according to your security mode. We provide two examples below, one for each security mode. Locate the sample appropriate to your security mode, copy the JSON, paste it into a new file, and save it as `config.json`.
+
+`strict` **mode**
+
+```json
+{
+  "service": {
+    "name": "jenkins",
+    "user": "nobody",
+    "security": {
+      "service-account": "jenkins-principal",
+      "secret-name": "jenkins/private_key",
+      "strict-mode": true
+    }
+  }
+}
+```
+
+`permissive` **mode**
+
+```json
+{
+  "service": {
+    "name": "jenkins",
+    "user": "nobody",
+    "security": {
+      "service-account": "jenkins-principal",
+      "secret-name": "jenkins/private_key",
+      "strict-mode": false
+    }
+  }
+}
+```
+
+If you have modified any of the values shown in the previous sections, change the values in the following JSON as appropriate.
+
+
+## Install Jenkins
+
+To install the service, complete the following command.
+
+```bash
+dcos package install --options=config.json beta-jenkins
+```
+---
+# Quota enforced Multi-Tenant Jenkins
+
+The DC/OS Jenkins service starting with DC/OS 2.0 supports multi-tenancy. Users should familiarize themselves with [Quota](../../../../2.0/multi-tenancy/quota-management) and [Resource Managment Primitives](../../../../2.0/multi-tenancy/resource-mgmt-primitives/) in DC/OS.
+
+**Note:** We will use `/dev/jenkins` as the name of the service for the quota enabled multi-tenant examples below.
+- DC/OS 2.0 and later clusters
+`dev` will be the group in which the Jenkins service will be deployed as well as the *role* which the service will use. In `strict` mode, the service will require permissions granted for this `dev` role.
+- DC/OS 1.13 and older clusters
+`dev` will be the group in which the Jenkins service will be deployed. The *role* used by the service can be configured with the default of `*`.  In `strict` mode, the service will require permissions granted for this `*` role.
+
+### Role used by Jenkins Agents
+- DC/OS 2.0 and later clusters - 
+When quota is enforced on the group, the Jenkins agents will inherit the same role as the Jenkins master.
```

Comment:
Really? Do you mean we set `JENKINS_AGENT_ROLE` appropriately?

Reply:
Yes we do set role correctly, 
See: https://github.com/mesosphere/dcos-jenkins-service/blob/4.x.y/scripts/dcos-quota.sh#L30-L40

`JENKINS_AGENT_ROLE` is an internal property and has less meaning to the end user compared to the docs statement above.